### PR TITLE
Changes to resolve a bug on formatting the metrics

### PR DESCRIPTION
**Pull request by jzhang-gp**
This is a temporary fix for the bug in production on model metrics. Since the metrics can be `None`, rounding the number could cause error. 

The change here is just a quick patch on the code. In the long run, I want to move the computation of the metrics from the frontend to the backend as well as refactoring the metrics formatting in a better way plus good tests...

### CODE REVIEW

File: db/model.py [hunk 1]
```diff
@@ -949,3 +958,35 @@ def load_inference(version_dir: str, dataset_filename: str,
...
+
+    for metric in test_metrics:
+        test_metrics[metric] = _format_float_numbers(test_metrics[metric])
+
+    logging.info(metrics)
+
+    return metrics
+
+
+def _format_float_numbers(nums):
```
Comment (ShahNewazKhan) [inline db/model.py:985]: Given this is a utility function not tied to _reformat_metrics() should we also parameterize the precision and set default to `2f` ?

Comment (jzhang-gp) [reply db/model.py]: > Given this is a utility function not tied to `_reformat_metrics()` should we also parameterize the precision and set default to `2f` ?

Updated the precision to a parameter.

Comment (ShahNewazKhan) [reply db/model.py]: :+1:   :shipit:

**Review by ShahNewazKhan — APPROVED** [review]
:shipit:

**Pull request merged**